cli/Tests/TuistGeneratorTests/Generator/BuildableFolderCheckerTests.swift

```diff
@@ -63,3 +65,34 @@ struct BuildableFolderCheckerTests {
         #expect(got == false)
     }
 }
+
+private let validResourceExtensions = [
+    // Regular resource files
+    "md",
+    "xcstrings",
+    "plist",
+    "rtf",
+    "tutorial",
+    "sks",
+    "xcprivacy",
+    "gpx",
+    "strings",
+    "stringsdict",
+    "geojson",
+    "storyboard",
+    "xib",
+    "xcfilelist",
+    "xcconfig",
+    "txt",
+    "json",
+    "js",
+
+    // Folder extensions that are added as resources
+    "xcassets",
+    "scnassets",
+    "bundle",
+    "xcstickers",
+    "app",
+    "xcmappingmodel",
+    "xcdatamodeld"
+]
```

Comment:
we're basically copying what we have in `XcodeGraph`. I'd include just the ones that were causing issues instead of having the same list in both places